cves/kernel/CVE-2015-8812.yml

```diff
@@ -34,11 +39,11 @@ announced_instructions: |
   This is not the same as published date in the NVD - that is below.
 
   Please enter your date in YYYY-MM-DD format.
-announced_date: '2016-04-27'
+announced_date: 2015-12-11
 published_instructions: |
   Is there a published fix or patch date for this vulnerability?
   Please enter your date in YYYY-MM-DD format.
-published_date: '2016-04-27'
+published_date: 2016-04-17 #published to stable - https://yhbt.net/lore/all/1460886804-19372-31-git-send-email-sasha.levin@oracle.com/
```

Comment:
I wouldn't say the comment here is necessary. Maybe if there was a notes tab for this section, but not as is.

Reply:
The comment is primarily for myself and future readers so they know where each piece of information came from. It should have no impact on the VHP entry.

Reply:
Read previous comment - https://github.com/VulnerabilityHistoryProject/vulnerabilities/pull/209#discussion_r1390334647